app/src/main/java/ch/epfl/sweng/swenggolf/database/FakeDatabase.java

```diff
@@ -71,4 +74,201 @@ public void write(@NonNull String path, @NonNull String id, @NonNull Object obje
         }
         return list.isEmpty() ? null : list;
     }
+
+    /**
+     * Creates a database already filled with users and offers.
+     *
+     * @return the prefilled database.
+     */
+    public static Database fakeDatabaseCreator() {
+        Database fakeValuedDatabase = new FakeDatabase(true);
+        for (User user : FAKE_USERS) {
+            fakeValuedDatabase.write("/users", user.getUserId(), user);
+        }
+        for (Offer offer : FAKE_OFFERS) {
+            fakeValuedDatabase.write("/offers", offer.getUuid(), offer);
+        }
+        return fakeValuedDatabase;
+    }
+
+    private static final String WIKIA_NO_COOKIE = "https://vignette.wikia.nocookie.net/";
+    private static final String LUMIERE = "https://lumiere-a.akamaihd.net/v1/images/";
+    private static final String STARWARS = "starwars/";
+
+    public static final User[] FAKE_USERS = {
```

Comment:
An idea to resolve this Code Climate issue could be to create one entire new class that creates the users, and another that creates the offers?

Reply:
I thought about it, but I wanted to have all the values inside the FakeDatabase, since it's only related to it. Plus, we might change from public to private. But I guess we don't have that much choice. Actually I just deleted some end lines.

Reply:
Or we could just tell Code Climate that 300 lines are still ok, it's easy to modify the config file.

Reply:
Otherwise I can create a subclass to FakeDatabase.